developing_packages/workspaces.rst

```diff
@@ -70,31 +69,30 @@ Next, run a :command:`conan install` as usual, using a *build* folder to output
     Workspace HelloA: Generated conaninfo.txt
     Workspace HelloA: Generated conanbuildinfo.txt
 
-
-Note that nothing will really be installed in the local cache. All the dependencies are resolved locally:
+Note that nothing will really be installed in the local cache, all the dependencies are resolved locally:
 
 .. code-block:: bash
 
     $ conan search
     There are no packages
 
-Also, all the generated *conanbuildinfo.cmake* files for the dependencies are installed in the *build* folder. You can inspect them to check
-that the paths they define for their dependencies are user folders. They don't point to the local cache.
+Also, all the generated *conanbuildinfo.cmake* for each dependencies are installed in the *build* folder. You can inspect them to check
```

Comment:
Each dependency generates one *conanbuildinfo.cmake* file, all those files are installed in the *build* folder. I think this is not what the sentence says.